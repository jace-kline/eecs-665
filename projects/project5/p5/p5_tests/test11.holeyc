int g() {
    return 5;
}

int main() {
    TOCONSOLE g;
    TOCONSOLE g();
    boolptr p;
    bool b;
    b = true && @p;
    return 0;
}