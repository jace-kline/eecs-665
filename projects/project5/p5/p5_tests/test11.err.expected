FATAL [6,15]: Attempt to output a function
FATAL [10,17]: Logical operator applied to non-bool operand
Type Analysis Failed

